cimgen/languages/modernpython/templates/cimpy_constants_template.mustache

```diff
@@ -1,6 +1,10 @@
+"""
+Generated from the CGMES files via cimgen: https://github.com/sogno-platform/cimgen
+"""
+
 # Default namespaces used by CGMES.
-NAMESPACES = {  # Those are strings, not real addresses, hence the NOSONAR.
-    "cim": "http://iec.ch/TC57/CIM100#",  # NOSONAR
+NAMESPACES: dict[str, str] = {  # Those are strings, not real addresses, hence the NOSONAR.
+    "cim": "{{cim_namespace}}",  # NOSONAR
```

Comment:
what about
```xmlns:eu="http://iec.ch/TC57/CIM100-European#"``` shouldn't we manage it as well?

Reply:
You are right. I haven't seen that there are some classes and attributes with a different namespace.
For CGMES 2.4 it's the same with namespace http://entsoe.eu/CIM/SchemaExtension/3/1#.
But there are only few of these classes or attributes. Do you know if these are used often?
For all other classes/attributes, i.e. for 99%, the serialisation is correct. So I think the writer could be used in the current state.
BTW: In the cimpy writer these classes/attributes is also not correctly serialized.

Reply:
I don't know by heart which attributes are CGMES (and hence with `eu` prefix) specific, but I can check. I suspect they are not many, but then judging their importance is not simple, because it depends on the importance for the user.

Reply:
from my point of view, this can be tackled in other PRs (and eventually we can cover it)

about writers, I think adding intelligence to the dataclass to serialize data, may be a good strategy.

Reply:
> I don't know by heart which attributes are CGMES (and hence with `eu` prefix) specific, but I can check. I suspect they are not many, but then judging their importance is not simple, because it depends on the importance for the user.

it boils down to 1 class in eq (boundarypoint), its attributes and few other attributes (e.g. OperationalLimitType.kind)